### PR TITLE
fix(mac): generate missing macOS asset catalog icons

### PR DESCRIPTION
**Pull request by hansemannn**
**JIRA:** https://jira.appcelerator.org/browse/TIMOB-28185

### CODE REVIEW

**Comment by rohidjetha** [comment]
Just tried it and it's working great!

**Review by sgtcoolguy — APPROVED** [review]
LGTM

File: iphone/cli/commands/_build.js [hunk 1]
```diff
@@ -5261,6 +5262,22 @@ iOSBuilder.prototype.copyResources = function copyResources(next) {
 					let defaultIconChanged = false,
 						defaultIconHasAlpha = false;
 
+					// Add macOS icons if target is macOS
+					if (this.target === 'macos') {
```
Comment (vijaysingh-axway) [inline iphone/cli/commands/_build.js:5266]: We should check for target 'dist-macappstore' as well ?
```suggestion
					if (this.target === 'macos' || this.target === 'dist-macappstore') {
```

**Comment by ssekhri** [comment]
FR Passed. MacOS icons added when build for target as 'macos' or 'dist-macappstore'.

Verified on:
Mac OS: 10.15.4
SDK: 9.3.0.v20201023101805
Appc CLI: 8.1.1
JDK: 11.0.4
Node: 10.17.0
Xcode: 12.0.1

**Pull request merged**